lib/views/background_sound/background_sound_view.dart

```diff
@@ -127,30 +134,36 @@ class _BackgroundSoundViewState extends ConsumerState<BackgroundSoundView> {
           ref.read(backgroundSoundsProvider);
         }
       },
-      child: CollapsibleHeaderWidget(
-        title: StringConstants.backgroundSounds,
-        leadingIconBgColor: ColorConstants.walterWhite,
-        leadingIconColor: ColorConstants.black,
-        headerHeight: 130,
-        children: [
-          VolumeSliderWidget(),
-          SoundListTileWidget(
-            sound: BackgroundSoundsModel(
-              id: '0',
-              title: StringConstants.none,
-              duration: 0,
-              path: '',
-            ),
+      child: CustomScrollView(
+        controller: _scrollController,
+        physics: const AlwaysScrollableScrollPhysics(),
+        slivers: [
+          MeditoAppBarLarge(
+            scrollController: _scrollController,
+            title: StringConstants.backgroundSounds,
           ),
-          Column(
-            children: data
-                .map((e) => SoundListTileWidget(
-                      sound: e,
-                    ))
-                .toList(),
+          SliverList(
+            delegate: SliverChildListDelegate([
+              VolumeSliderWidget(),
+              SoundListTileWidget(
+                sound: BackgroundSoundsModel(
+                  id: '0',
+                  title: StringConstants.none,
+                  duration: 0,
+                  path: '',
```

Comment:
Is it possible for these values to be defaults? @osamaasifoak  I was a bit confused seeing the 0s and empty strings

Reply:
Fixed!!